### ISSUE TITLE
[Android]: add Overlay Editor in On-Screen Display -> On-Screen Overlay

### ISSUE THREAD

**Issue by ghost** [libretro/RetroArch#14152]
*Overlay Editor* means moving **D-Pad,** moving **Buttons (Y, X, B, A),** moving **L1, R1, L2, R2, L3, R3** and Resize **D-Pad,** **Buttons (Y, X, B, A),** **L1, R1, L2, R2, L3, R3** (10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 100%, 110%, 120%, 130%, 140%, 150%, 160%, 170%, 180%, 190%, 200% Resize **D-Pad,** **Buttons (Y, X, B, A),** **L1, R1, L2, R2, L3, R3).** The default overlay size is 50%

**Comment by naxil** [comment]
yes for my opinion its good idea. on possibility for bind with virtual pad (es= bind AI button or add it)
